engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -1695,6 +1607,42 @@ public void run()
             }
             endWindowStatsMap.put(shb.getNodeId(), endWindowStats);
 
+            if (!oper.getInputs().isEmpty()) {
+              long latency = Long.MAX_VALUE;
```

Comment:
@davidyan74 : Do we need to care about following scenario
Operator C has two upstream A and B. Current latency of A > B and based on that C adds A as it's slowest upstream but then B's stats come and now latency of B > A but C is still pointing A as it's slowest upstream...